modules/programs/direnv.nix

```
@@ -162,28 +162,20 @@ in {
           $env.config.hooks.pre_prompt?
           | default []
           | append {||
-              let direnv = (
-                  ${direnvWrapped}
-                  | from json --strict
-                  | default {}
-              )
-              if ($direnv | is-empty) {
-                  return
-              }
-              $direnv
+              ${direnvWrapped}
+              | from json --strict
+              | default {}
               | items {|key, value|
-                  {
-                      key: $key
-                      value: (do (
-                          $env.ENV_CONVERSIONS?
-                          | default {}
-                          | get -i $key
-                          | get -i from_string
-                          | default {|x| $x}
-                      ) $value)
-                  }
+                let value = do (
+                  $env.ENV_CONVERSIONS?
+                  | default {}
+                  | get -i $key
+                  | get -i from_string
+                  | default {|x| $x}
+                ) $value
```

Indentation goes from 4 spaces to 2 spaces over here

```suggestion
                  let value = do (
                      $env.ENV_CONVERSIONS?
                      | default {}
                      | get -i $key
                      | get -i from_string
                      | default {|x| $x}
                ) $value
```